yt_dlp/extractor/mave.py

```diff
@@ -105,3 +106,53 @@ def _real_extract(self, url):
                 'uploader': ('author', {str}),
             })),
         }
+
+
+class MaveChannelIE(InfoExtractor):
+    IE_NAME = 'mave:channel'
+    _VALID_URL = r'https?://(?P<id>[\w-]+)\.mave\.digital/?$'
+    _TESTS = [{
+        'url': 'https://budem.mave.digital/',
+        'info_dict': {
+            'id': 'budem',
+            'title': 'Все там будем',
+            'description': 'md5:f04ae12a42be0f1d765c5e326b41987a',
+        },
+        'playlist_mincount': 15,
+    }, {
+        'url': 'https://ochenlichnoe.mave.digital/',
+        'info_dict': {
+            'id': 'ochenlichnoe',
+            'title': 'Очень личное',
+            'description': 'md5:ee36a6a52546b91b487fe08c552fdbb2',
+        },
+        'playlist_mincount': 20,
+    }]
+
+    def _real_extract(self, url):
+        channel_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, channel_id)
+
+        nuxt_data = self._search_nuxt_json(webpage, channel_id)
+
+        channel_title = traverse_obj(nuxt_data, (
+            'pinia', 'common', 'podcast', 'podcast', 'title'))
+        channel_description = traverse_obj(nuxt_data, (
+            'pinia', 'common', 'podcast', 'podcast', 'description'))
```

Comment:
can be inlined as a single traverse_obj call that is unpacked in to kwargs of `playlist_result`